app/src/main/java/com/kickstarter/models/CommentThread.java

```diff
@@ -0,0 +1,67 @@
+package com.kickstarter.models;
+
+import android.os.Parcelable;
+import androidx.annotation.Nullable;
+import com.kickstarter.libs.qualifiers.AutoGson;
+import org.joda.time.DateTime;
+
+import java.util.List;
+import java.util.Objects;
+import auto.parcel.AutoParcel;
+
+@AutoGson
+@AutoParcel
+public abstract class CommentThread implements Parcelable{
```

Comment:
Is this `CommentThread` object only for threaded comments? or does it also represent root comments? If it represents both, we should rename this as it is a bit confusing otherwise.

Reply:
there is another model called comment. replies and (root)comment share same model

Reply:
@sunday-okpoluaefe in this PR already merged https://github.com/kickstarter/android-oss/pull/1247 @leighdouglas has renamed the old comment related classes. You can name this model as `Comment` without any conflict

Reply:
ok great

Reply:
renamed to ``` Comment ```